modules/islandora_crossref_citations/README.md

```diff
@@ -0,0 +1,49 @@
+# Islandora Crossref Citations
+
+## Introduction
+
+Islandora Crossref Citations generates a block which displays citation counts for any object with a DOI, via the Crossref Cited-by service, for use with Islandora Badges.
+
+The badge will only display on objects with a MODS datastream and a DOI (Digital Object Identifier).
+
+## Requirements
+
+This module requires the following modules/libraries:
+
+* [Islandora](https://github.com/islandora/islandora)
+* [Islandora Scholar](https://github.com/Islandora/islandora_scholar)
+* [Islandora Badges](../../)
```

Comment:
Since this is a submodule (in terms of publishing) but is just a module for Drupal's eyes, it seems to me that using relative paths in the documentation is very context dependent. .md files should be readable outside the github context.